global.json

```diff
@@ -0,0 +1,5 @@
+{
+  "sdk": {
+    "version": "3.1.101"
```

Comment:
Can you put the same version as in the build.config?

Reply:
I think the artifacts test that test against 2.1 will fail because of this file, I remember I removed it because of that. I added a global json in the parent folder that contains my local copy of the repo